packages/react-native-payments/lib/ios/ReactNativePayments.m

```diff
@@ -90,6 +90,13 @@ - (NSDictionary *)constantsToExport
     callback(@[[NSNull null]]);
 }
 
+RCT_EXPORT_METHOD(openPaymentSetup: (RCTResponseSenderBlock)callback) {
+    PKPassLibrary* lib = [[PKPassLibrary alloc] init];
+    [lib openPaymentSetup];
+
+    callback(@[[NSNull null]]);
```

Comment:
@chukohsin 

Seems like you can use that callback function to use data from the response:
```
ApplePaySession.openPaymentSetup(merchantIdentifier)
.then(function(success) {
    if (success) {
        // Open payment setup successful
    } else {
        // Open payment setup failed
    }
}).catch(function(e) {
    // Open payment setup error handling
});
```

Reply:
Same as the above comment.